### PR TITLE
added path in master (maybe successfully??)

### PR DESCRIPTION
**Pull request by m-w-andrews**

### CODE REVIEW

File: _ethiopia_ie_master.R [hunk 1]
```diff
@@ -11,6 +11,8 @@
 if(Sys.info()[["user"]] == "WB554990") project_file_path <- "C:/Users/wb521633/Dropbox/World Bank/IEs/Ethiopia IE - Merge Budget Data With Shapefile"
 if(Sys.info()[["user"]] == "WB521633") project_file_path <- "C:/Users/wb521633/Dropbox/World Bank/IEs/Ethiopia IE"
 if(Sys.info()[["user"]] == "robmarty") project_file_path <- "~/Dropbox/World Bank/IEs/Ethiopia IE"
+if(Sys.info()[["user"]] == "andm2") project_file_path <- "C:/Users/andm2/Dropbox/WorldBank/Ethiopia IE"
+
 
 if(Sys.info()[["user"]] == "robmarty") code_file_path <- "~/Documents/Github/Ethiopia-Corridors-IE"
```
Comment (ramarty) [inline _ethiopia_ie_master.R:18]: @m-w-andrews Just for the sake of github practice, could you make another commit adding a path for `code_file_path`?

Once you do this, you'll see that the pull request automatically updates with this new commit. So when I then approve the pull request, _both_ commits will then be integrated into the master branch.

**Comment by ramarty** [comment]
@m-w-andrews Perfect! Just for the sake of github practice, could you make another commit adding a path for code_file_path?

Once you do this, you'll see that the pull request automatically updates with this new commit. So when I then approve the pull request, both commits will then be integrated into the master branch.

**Comment by m-w-andrews** [comment]
> @m-w-andrews Perfect! Just for the sake of github practice, could you make another commit adding a path for `code_file_path`?
> 
> Once you do this, you'll see that the pull request automatically updates with this new commit. So when I then approve the pull request, both commits will then be integrated into the master branch.

Done!
